tests-beta/android/MASVS-STORAGE/MASTG-TEST-0202.md

```diff
@@ -17,7 +17,7 @@ This static test is great for identifying all code locations where the app is wr
 ## Steps
 
 1. Reverse engineer the app (@MASTG-TECH-0017).
```

Comment:
MASTG-TECH-0017 is meant for when you have the original code.

We need to use **MASTG-TECH-0013** (Reverse Engineering Android Apps) and its sub-techniques MASTG-TECH-0016, MASTG-TECH-0017, MASTG-TECH-0018.